src/Storages/RabbitMQ/StorageRabbitMQ.cpp

```diff
@@ -954,8 +954,10 @@ bool StorageRabbitMQ::checkDependencies(const StorageID & table_id)
 {
     // Check if all dependencies are attached
     auto view_ids = DatabaseCatalog::instance().getDependentViews(table_id);
+    LOG_TEST(log, "Number of attached views {} for {}", view_ids.size(), table_id.getNameForLogs());
+
     if (view_ids.empty())
-        return true;
+        return false;
```

Comment:
looks like we return true if table/view has dependencies, else we return false. Maybe we can change the function name to `hasDependencies`.